pages/common/gh-browse.md

```diff
@@ -21,15 +21,15 @@
 
 - Open a specific issue or pull request in the web browser:
 
-`gh browse {{issue_or_pull_request_number}}`
+`gh browse {{issue_identifier|pull_request_identifier}}`
```

Comment:
I prefer `number` over ID. Also a single placeholder sounds easier to read.

Reply:
We can require ID just if it is not a plain integer. In this case I can use `number` then. :)